src/sentry/api/serializers/models/team.py

```diff
@@ -16,48 +17,55 @@
     ProjectTeam,
     Team,
     TeamAvatar,
+    User,
 )
 from sentry.utils.compat import zip
 
 
-def get_team_memberships(team_list, user):
+def get_team_memberships(team_list: Sequence[Team], user: User) -> Set[int]:
```

Comment:
Generally speaking I think runtime behavior should not be changed because of some types. It seems better to return `Iterable[int]` here and `# type: ignore` in function body if necessary for django

Reply:
I agree. If we bury functional changes in typing PRs we are opening ourselves up to bugs.